app/main.py

```diff
@@ -38,3 +38,18 @@ async def index():
 def blocking():
     time.sleep(2)
     return {"ret": 42}
+
+# Get https://phinvads.cdc.gov/baseStu3/CodeSystem/ and return a whole code system result
+@app.get("/phinvads/CodeSystem")
+@cache(namespace="phinvads", expire=3600)
+def get_code_systems(code: str = None):
+    url = "https://phinvads.cdc.gov/baseStu3/CodeSystem"
+    params = {
+        "code": code,
+    }
+    response = requests.get(url, params=params)
```

Comment:
can update to use `get()` from Nick's PR: https://github.com/CDCgov/vocabulizer/blob/main/app/main.py#L76